docs/apidocs/qiskit_addon_obp.utils.simplify.rst

```diff
@@ -1,10 +1,14 @@
-﻿========
-simplify
-========
-
-.. _qiskit_addon_obp-utils-simplify:
+﻿======================================================================
+Pauli operator simplification (:mod:`qiskit_addon_obp.utils.simplify`)
+======================================================================
 
 .. automodule:: qiskit_addon_obp.utils.simplify
    :no-members:
    :no-inherited-members:
    :no-special-members:
+
+.. currentmodule:: qiskit_addon_obp.utils.simplify
+
+.. autoclass:: OperatorBudget
+.. autoclass:: SimplifyMetadata
+.. autofunction:: simplify
```

Comment:
In contrast, this here places all the classes and functions on a single page, thereby not using the `class.rst` template and, thus, not using the "defaults" which we have encoded there e.g. the `:show-inheritance:` setting..